docs/userguides/accounts.md

```diff
@@ -66,6 +66,23 @@ ape accounts import <ALIAS>
 It will prompt you for the private key.
 If you need help exporting your private key from Metamask, see [this guide](https://metamask.zendesk.com/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key).
 
+You can also import accounts from mnemonic seed by using the `--mnemonic` flag:
+
+```bash
+ape accounts import <ALIAS> --mnemonic
+```
+
+It will then prompt you for the mnemonic seed. If you need help finding your mnemonic seed (Secret Recovery Phrase) in Metamask, see [this guide](https://metamask.zendesk.com/hc/en-us/articles/360015290032-How-to-reveal-your-Secret-Recovery-Phrase). 
+
+In addition, you can also use a custom HDPath by using the `--hdpath` flag:
+
+```bash
+ape accounts import <ALIAS> --mnemonic --hdpath
```

Comment:
I'm having trouble using `--hdpath` when not a flag since `m/44'/61'/0'/0/0` needs to be wrapped in quotes `"m/44'/61'/0'/0/0"` or it will bother the bash cli.

Reply:
Wrapping in quotes is fine.

Reply:
It what users who use `ape-trezor` and `ape-ledger` have to do. It is normal in CLI applications.